network-api/networkapi/buyersguide/factory.py

```diff
@@ -111,18 +112,49 @@ def set_image(self, create, extracted, **kwargs):
 
     worst_case = Faker('sentence')
 
+    signup_requires_email = LazyFunction(get_extended_yes_no_value)
+    signup_requires_phone = LazyFunction(get_extended_yes_no_value)
+    signup_requires_third_party_account = LazyFunction(get_extended_yes_no_value)
+    signup_requirement_explanation = Faker('sentence')
+
+    meets_minimum_security_standards = Faker('boolean')
+    show_ding_for_minimum_security_standards = Faker('boolean')
+    uses_encryption = LazyFunction(get_extended_yes_no_value)
+    uses_encryption_helptext = Faker('sentence')
+    security_updates = LazyFunction(get_extended_yes_no_value)
+    security_updates_helptext = Faker('sentence')
+    strong_password = LazyFunction(get_extended_yes_no_value)
+    strong_password_helptext = Faker('sentence')
+    manage_vulnerabilities = LazyFunction(get_extended_yes_no_value)
+    manage_vulnerabilities_helptext = Faker('sentence')
+    privacy_policy = LazyFunction(get_extended_yes_no_value)
+    privacy_policy_helptext = Faker('sentence')
+
+    @post_generation
+    def set_privacy_policy_link(self, create, extracted, **kwargs):
+        ProductPrivacyPolicyLinkFactory.create(product=self)
+
+    phone_number = Faker('phone_number')
+    live_chat = Faker('url')
+    email = Faker('email')
+
+    if random() > 0.5:
+        twitter = '@TwitterHandle',
+
+    # updates...
+
+
+class GeneralProductFactory(ProductFactory):
```

Comment:
I've made `ProductFactory` the superclass for the general/software factories, similar to how general/software products have a base class.

Reply:
line 162-163 should be moved to the super class
```py
how_does_it_use_data_collected = Faker('sentence')
data_collection_policy_is_bad = Faker('boolean')
```